include/mbedtls/psa_util.h

```diff
@@ -235,7 +235,85 @@ static inline int mbedtls_psa_get_ecc_oid_from_id(
      return( -1 );
 }
 
-#define MBEDTLS_PSA_MAX_EC_PUBKEY_LENGTH 256
+#define MBEDTLS_PSA_MAX_EC_PUBKEY_LENGTH 1
```

Comment:
Thanks for fixing this. We should probably ask if this should be provided by PSA, as I think they have a policy to provide macros for the maxiumx size of each possible output buffers to their functions, which I think could cover this (or close enough that we don't have to redo everything from scratch here.)

(This should not block this PR, just making a note for the future.)